### ISSUE TITLE
Custom module disappears after performing click action

### ISSUE THREAD

**Issue by GSF1200S** [Alexays/Waybar#379]
I have a custom module that shows Rhythmbox data (Song: Artist), and that executes play/pause when I click it. Up until recently it was fine. Now whenever I click the module to play/pause, it executes the command properly but disappears from Waybar. This is the snippet from my config:

`"custom/rhythmbox": {
        "format": " ♪  {}",
        "max-length": 50,
        "interval": 10,
        "tooltip": false,
        "exec": "rhythmbox-client --print-playing-format=\"%tt:  %aa\"",
        "on-click": "rhythmbox-client --play-pause"
        //"exec-if": "pgrep spotify"
    }`

Running Waybar from the CLI yields no output relating to the crashed module. If I try waybar -l debug or waybar --log-level debug as per --help, it tells me that these are unknown options. Waybar is installed on an up-to-date Arch install using the package from Community. Anything else I can provide to help let me know...

**Comment by Alexays** [comment]
Weird, i can't reproduce even using your custom module

**Comment by SlavojZizekIsASnack** [comment]
I get the same issue. Mwe: 
```
"custom/cmus": {
	"format": "{} ♪",
	"interval": 1,
	"exec": "echo \"asdasd\"",
	"on-click": "cmus-remote -C player-pause"
}
```
The problem disappears if you remove `"on-click"`.

**Comment by GSF1200S** [comment]
> I get the same issue. Mwe:
> 
> ```
> "custom/cmus": {
> 	"format": "{} ♪",
> 	"interval": 1,
> 	"exec": "echo \"asdasd\"",
> 	"on-click": "cmus-remote -C player-pause"
> }
> ```
> 
> The problem disappears if you remove `"on-click"`.

I forgot to mention this in my OP but I have a dual screen setup with Waybar on both monitors- do you have a dual monitor setup as well? Thats the only thing I can think of that *might* be different than the setup @Alexays is using...

**Comment by SlavojZizekIsASnack** [comment]
Nope : /

**Comment by Alexays** [comment]
Fixed with 564fdcb3693786234608a0fab7063ccba5388e10

**Comment by SlavojZizekIsASnack** [comment]
Fix works great, thank you Alexays :)
